xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1281,8 +1338,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value indicating whether the group collection has a fixed size.</summary>
+        <value><see langword="true" /> always.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. ```

Reply:
Same with the CDATA:

```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[  

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. 

           ]]></format>
         </remarks>
```